lib/signaling/v2/peerconnection.js

```diff
@@ -474,7 +483,7 @@ class PeerConnectionV2 extends StateMachine {
   _handleIceConnectionStateChange() {
     const { iceConnectionState } = this._peerConnection;
 
-    this._log.debug(`ICE connection state is "${iceConnectionState}"`);
+    this._log.debug(`ICE connection state is "${this._lastIceConnectionState}" => "${iceConnectionState}"`);
```

Comment:
@makarandp0 I think we should not change the log statement here since `_lastIceConnectionState` can be null.